data/infra/resources/chocolatey_config.yaml

```
@@ -41,7 +41,18 @@ properties_list:
   required: false
   description_list:
   - markdown: The value to set.
-examples: "**Set the Chocolatey cacheLocation config**:\n\n```ruby\nchocolatey_config
-  'Set cacheLocation config' do\n  config_key 'cacheLocation'\n  value 'C:\temp\boco'\nend\n```\n\n**Unset
-  a Chocolatey config**:\n\n```ruby\nchocolatey_config 'BogusConfig' do\n  action
-  :unset\nend\n```\n"
\ No newline at end of file
+examples: |
+  **Set the Chocolatey cacheLocation config**:
+  ```
+  chocolatey_config 'Set cacheLocation config' do
+    onfig_key 'cacheLocation'
+    value 'C:\temp\boco'
+  end
+  ```
+  **Unset a Chocolatey config**:
+  
+  ```
+  chocolatey_config 'BogusConfig' do
+    action :unset
+  end
+  ```
```

````suggestion
  **Set the Chocolatey cacheLocation config**:

  ```ruby
  chocolatey_config 'Set cacheLocation config' do
    config_key 'cacheLocation'
    value 'C:\temp\choco'
  end
  ```

  **Unset a Chocolatey config**:

  ```ruby
  chocolatey_config 'BogusConfig' do
    action :unset
  end
  ```
````

---

The examples aren't quite correct ☝️